packages/loading/src/createLoadingComponent.ts

```diff
@@ -87,31 +92,49 @@ export function createLoadingComponent({
 
       const spinnerText = h('p', { class: 'el-loading-text' }, [this.text])
 
+      const loadingContent = h('div', {
+        class: 'el-loading-spinner',
+      }, [
+        !this.spinner ? spinner : noSpinner,
```

Comment:
Can you change the order for better reading?

```suggestion
        this.spinner ? noSpinner : spinner,
```